packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -20,7 +20,9 @@ export interface ReplicaConfiguration {
   rumEndpointBuilder: EndpointBuilder
 }
 
-export function computeTransportConfiguration(initConfiguration: InitConfiguration): TransportConfiguration {
+export function computeTransportConfiguration(
+  initConfiguration: InitConfiguration & { usePciIntake?: boolean }
```

Comment:
**💬 suggestion:** ‏Maybe we could use `& { usePciIntake?: boolean }` only in the function it's used (`buildEndpointHost`).

It would be less explicit but at the same time a bit cleaner.

An alternative would be to define a different type for this configuration object, but I don't think it's worth it.

Reply:
Done ✔️